enterprise/backend/test/metabase_enterprise/sandbox/api/permissions_test.clj

```diff
@@ -44,12 +44,6 @@
                 :expected-perms   (fn []
                                     {:schemas {:PUBLIC {(mt/id :users) "all"}}})}
 
-               "when revoking segmented query permissions for Table"
-               {:updated-db-perms (fn []
-                                    {:native :none, :schemas {:PUBLIC {(mt/id :venues) {:read :all}}}})
-                :expected-perms   (fn []
-                                    {:schemas {:PUBLIC {(mt/id :venues) {:read "all"}}}})}
```

Comment:
Removed this test case because this isn't how sandboxed perms work anymore, at least not via the UI. "Segmented query perms" are just normal data access perms + an entry in the `sandboxes` table. 

There still are `/query/`, `/read/`, and `/query/segmented/` paths hanging around in app DBs, but we're not doing any enforcement based off of them. And they'll be removed whenever we migrate to a new schema.